SOWFA/postProcessing/averaging.py

```diff
@@ -1156,293 +1156,6 @@ def to_netcdf(self,fname):
 
 """end of class PlanarAverages"""
 
-class PlanarAverageFunctionObject(object):
```

Comment:
The function object is only going to be implemented within SOWFA6, right?

Reply:
That is correct